src/screens/AppIntroScreen.tsx

```diff
@@ -158,69 +246,60 @@ export const AppIntroScreen = ({ setOnboardingComplete, onlyTermsAndConditions }
   }
 
   return (
-    <SafeAreaViewFlex style={styles.background}>
+    <SafeAreaViewFlex style={[styles.background, { backgroundColor }]}>
       <StatusBar style="dark" translucent backgroundColor="transparent" />
       <AppIntroSlider<AppIntroSlide>
         activeDotStyle={onlyTermsAndConditions ? styles.hiddenDot : styles.activeDot}
+        bottomButton
         data={slides}
         dotClickEnabled={false}
         dotStyle={onlyTermsAndConditions ? styles.hiddenDot : styles.inactiveDot}
         keyExtractor={keyExtractor}
         onDone={setOnboardingComplete}
-        onSlideChange={(_, index) => {
-          // hide the next button on the terms and conditions slide
-          if (
-            index + 1 ==
-            slides.findIndex((slide) => slide.onLeaveSlideName === Initializer.TermsAndConditions)
-          ) {
-            setShowNextButtonTermsAndConditions(false);
-          }
-
-          slides[index]?.onLeaveSlide?.(true);
+        onSlideChange={handleSlideChange}
+        renderDoneButton={() => <SliderButton label={texts.appIntro.continue} isLightest />}
```

Comment:
why is this always `lightest` now?

Reply:
This actually changes the color of the text inside the button. I updated it to `isLightestText` to increase understandability